src/app/decks/Decks.js

```diff
@@ -123,6 +156,22 @@ class Decks extends Component {
             {activeTab === TABS.USER ? <SkillProgress decks={filteredDecks} /> : <HabitTracker />}
           </div>
         </div>
+        <div 
+          className="d-flex justify-content-center" 
+          onKeyPress={this.onKeyStroke}
+          tabIndex="0">
+            <Typeahead
+              className="search-term border-primary"
+              options={filteredNames}
+              onChange={(e) => {
+                this.setState({searchString: e[0]});
+              }}
+              placeholder="Search..."
+            />
+            <button type="submit" className="search-button text-white bg-primary border-info" onClick={this.handleSearch}>
+                <i className="fa fa-search"></i>
+            </button>
+        </div>
```

Comment:
Can you make the Searchbar into its own component? Would help with reuse in the future.